src/controllers/LocationController.js

```diff
@@ -0,0 +1,34 @@
+import dotenv from 'dotenv';
+import customize from '../helpers/Customize';
+import { cities } from '../database/models';
+
+dotenv.config();
+/**
+ * @export
+ * @class TripController
```

Comment:
It should be `@class LocationController` as opposed to write `@class TripController`